src/act.py

```
@@ -640,6 +671,257 @@ def delete_work(name, force):
+    )
+
+    # Display available works for selection
+    works = session.query(Work.id, Work.name, Work.description).all()
+    if not works:
+        console.print(
+            "[bold red]Error: No works found. Please create a work first.[/bold red]"
+        )
+        return
+
+    console.print("[bold green]Select a work for this assignment:[/bold green]")
+    for idx, (work_id, work_name, work_desc) in enumerate(works, 1):
```

Ruff (0.8.2)

Loop control variable `work_id` not used within loop body

(B007)